aspnetcore/migration/60-70.md

```diff
@@ -96,6 +96,17 @@ For more information, see <xref:blazor/js-interop/import-export-interop>.
 
 -->
 
+### .NET WebAssembly build tools for .NET 6 projects
+
+You can now use the .NET WebAssembly build tools with a .NET 6 project when working with the .NET 7 SDK. The new `wasm-tools-net6` workload includes the .NET WebAssembly build tools for .NET 6 projects so that they can be used with the .NET 7 SDK. The existing `wasm-tools` workload installs the .NET WebAssembly build tools for .NET 7 projects. However, the .NET 7 version of the .NET WebAssembly build tools are incompatible with existing projects built with .NET 6. Projects using the .NET WebAssembly build tools that need to support both .NET 6 and .NET 7 must use multi-targeting.
+
+New features in the `wasm-tools` workload for .NET 7 that help improve performance:
```

Comment:
This part highlights new features that should still be in the what's new section under a heading like: "Improvements to the .NET WebAssembly build tools" or something like that.